src/code/z_bgcheck.c

```diff
@@ -2913,25 +2913,25 @@ void DynaPoly_ExpandSRT(PlayState* play, DynaCollisionContext* dyna, s32 bgId, s
     }
 }
 
-void func_8003F8EC(PlayState* play, DynaCollisionContext* dyna, Actor* actor) {
+void DynaPoly_ResetInteractFlagsIfBgActor(PlayState* play, DynaCollisionContext* dyna, Actor* actor) {
     DynaPolyActor* dynaActor;
     s32 i;
 
     for (i = 0; i < BG_ACTOR_MAX; i++) {
-        if (dyna->bgActorFlags[i] & 1) {
+        if (dyna->bgActorFlags[i] & BG_DYNA_ACTOR_FLAG_IS_USED) {
             dynaActor = DynaPoly_GetActor(&play->colCtx, i);
             if (dynaActor != NULL && &dynaActor->actor == actor) {
-                func_800434A0((DynaPolyActor*)actor);
+                DynaPoly_ResetInteractFlags((DynaPolyActor*)actor);
                 return;
             }
         }
     }
 }
 
 /**
- * DynaPolyInfo_setup
+ * Original name: "DynaPolyInfo_setup"
  */
-void DynaPoly_Setup(PlayState* play, DynaCollisionContext* dyna) {
+void DynaPoly_UpdateContext(PlayState* play, DynaCollisionContext* dyna) {
```

Comment:
```suggestion
void DynaPoly_Update(PlayState* play, DynaCollisionContext* dyna) {
```
'Context' seems unnecessary to add

Reply:
when looking at this single function yeah, but "update dynapoly" could be many things that this is not imo (like, refreshing the lookup maybe?)